source/faas-automated-testing/faas-automated-testing.rst

```diff
@@ -267,6 +267,8 @@ variables. An explanation of the required environment is as follows:
 +-------------------------------+-------------------------------------+
 | AWS_SESSION_TOKEN             | Assume role automatically sets this |
 +-------------------------------+-------------------------------------+
+| MONGODB_VERSION               | The major version of the cluster    |
++-------------------------------+-------------------------------------+
 
```

Comment:
```suggestion
```

The version defaults in drivers-evergreen-tools, should we rely on the default so that drivers test consistently against the same server version?

Reply:
Good point, removed.